### PR TITLE
Switch to deb distribution, update to 2.2.0RC3

### PR DESCRIPTION
**Pull request by bobrik**
`1.397GB` -> `381MB` :tada:

### CODE REVIEW

**Comment by bobrik** [comment]
As usual, send your feedback, I can merge myself :octocat:

**Comment by abraithwaite** [comment]
:rocket:

File: Dockerfile [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
 RUN useradd opentsdb && \
     apt-get update && \
     apt-get upgrade -y && \
     apt-get install --no-install-recommends -y gnuplot && \
     apt-get clean && \
-    curl -sL https://github.com/OpenTSDB/opentsdb/releases/download/v${OPENTSDB_VERSION}/opentsdb-${OPENTSDB_VERSION}.tar.gz | tar zx -C /opt && \
-    mv /opt/opentsdb-${OPENTSDB_VERSION} /opt/opentsdb && \
-    cd /opt/opentsdb && \
-    apt-get install -y make && \
-    sed -i "s/MAX_NUM_TAGS = 8/MAX_NUM_TAGS = 12/" src/core/Const.java && \
```
Comment (dqminh) [inline Dockerfile:14]: does this change preserve the MAX_NUM_TAGS change we had before ?

Comment (bobrik) [inline Dockerfile:14]: `tsd.storage.max_tags` is available upstream now.

**Pull request merged**